docs/admin/machine.rst

```diff
@@ -87,10 +88,10 @@ DeepL
 
 .. versionadded:: 2.20
 
-DeepL is paid service providing good machine translation for few languages.
-According to some benchmark it's currently best available service.
```

Comment:
What benchmarks are these?

Reply:
This was based on internal benchmarking done by one of Weblate customers. But that's at least two years old, so it probably needs updating and up to date references.